marshal_3_smallint_test.go

```diff
@@ -14,70 +15,77 @@ func TestMarshalSmallint(t *testing.T) {
 		return Unmarshal(NativeType{proto: 4, typ: TypeSmallInt}, bytes, i)
 	}
 
-	brokenTypes := utils.GetTypes(mod.String(""), (*mod.String)(nil))
+	brokenMarshalTypes := utils.GetTypes(mod.String(""), (*mod.String)(nil), big.Int{}, &big.Int{})
```

Comment:
Can you please rename this test from `TestMarshalSmallint` to `TestMarshalSmallInt`